src/plugins/console/server/lib/spec_definitions/json/generated/clear_scroll.json

```diff
@@ -6,6 +6,6 @@
     "patterns": [
       "_search/scroll"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/search-request-body.html#_clear_scroll_api"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/ux/#scroll-search"
```

Comment:
I think any of these spec_definitions APIs that don't have documentation in the `/rest-api` directory should be noted in `noDocumentation` because not all parameters and details are explained in the other pages (except cases like the [Index aliases page](https://opensearch.org/docs/latest/opensearch/index-alias/)).